build.gradle

```diff
@@ -83,6 +86,18 @@ sourceSets {
     }
 }
 
+sigstoreSign {
+    oidcClient {
+        gitHub {
+            audience.set("sigstore")
```

Comment:
This adds the [Sigstore](https://sigstore.dev) plugin for Gradle. I went back and forth on this one, because I don't see any publishing steps here for libraries, so I figure it is intended as an app module.

I added it in anyway because, if a downstream user _does_ use this sample to publish a library, they will get a smooth experience which is enabled out of the box for provenance-enabled publishing.